invenio_app_ils/ui/backoffice/src/common/api/item.js

```diff
@@ -2,15 +2,33 @@ import { http } from './base';
 
 const itemURL = '/items/';
 
-const getList = () => {
-  return http.get(itemURL);
+const getRecord = itemPid => {
+  return http.get(`${itemURL}${itemPid}`);
 };
 
-const getRecord = itemId => {
-  return http.get(`${itemURL}${itemId}`);
+const mapStatus = s => {
```

Comment:
IMHO, REST API should never change data, because you don't know which client is going to consume it.
It is exactly the responsibility of the client to present data in human readable format, depending on the context (users, timezone, device, etc...)

Reply:
removed mappings